src/components/progress-tracker/progress-tracker.style.js

```diff
@@ -63,7 +62,9 @@ const StyledValuesLabel = styled.span`
   text-align: start;
   display: flex;
   justify-content: space-between;
-  ${({ isVertical, position }) => css`
+  ${({ isVertical, position, size }) => css`
```

Comment:
```suggestion
  font: ${getTypographyToken(size)};
  ${({ isVertical, position }) => css`
```

Looks good to me! `size` can be taken out of this block here, but this is really minor thing ✅